test/utils/InstanceTest.sol

```diff
@@ -86,7 +86,7 @@ abstract contract InstanceTest is HyperdriveTest {
         super.setUp();
 
         // Initial contribution.
-        uint256 contribution = 5_000e18;
```

Comment:
These changes aren't going to work for the other yield sources. What I think would fix all of these issues would be to pass in the decimals as part of the config, and then we could calculate the contribution amounts as:

```
uint256 contribution = 5_000 * 10 ** config.decimals;
```